### ISSUE TITLE
Accessibility labels missing on add credentials modal

### ISSUE THREAD

**Issue by jleach** [bcgov/bc-wallet-mobile#1322]
The questions marks near accessibility labels often means they are missing and the operation system, in this case iOS is guessing them. We should fix this by explicitly assigning accessibility labels.

![IMG_0064.PNG](https://images.zenhubusercontent.com/5c2fa1b7ce4f66381cd8d02d/39719ed3-e697-4910-81fa-71aabed85ff1)

**Comment by jleach** [comment]
Verified in 1.0.15-1452
